pkgs/tools/misc/parallel/default.nix

```diff
@@ -12,10 +14,12 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ makeWrapper ];
   buildInputs = [ perl procps ];
+  propagatedbuildInputs = extraPerlPackages;
 
-  postInstall = ''
+  postInstall = with perlPackages; ''
     wrapProgram $out/bin/parallel \
-      --prefix PATH : "${stdenv.lib.makeBinPath [ procps perl ]}"
+      --prefix PATH : "${stdenv.lib.makeBinPath [ procps perl ]}" \
+      --set PERL5LIB "${perlPackages.makeFullPerlPath extraPerlPackages}"
```

Comment:
Since it is just an environment variable added I would make `parallel-full` a wrapper around `parallel` that adds these packages.

Reply:
Not 100% sure if the wrapper approach i used is correct, but it works. Open to a better approach.